### ISSUE TITLE
Broken link for "discussion in Alex Krizhevsky's cuda-convnet library API"

### ISSUE THREAD

**Issue by liviu-** [cs231n/cs231n.github.io#112]
[Here](https://github.com/cs231n/cs231n.github.io/blob/master/convolutional-networks.md#normalization-layer) it is linked to [this wiki page](http://code.google.com/p/cuda-convnet/wiki/LayerParams#Local_response_normalization_layer_(same_map)) which is not working anymore, and does not appear to have a cached version online as verified with [this service](http://cachedview.com/). The [wiki](https://code.google.com/p/cuda-convnet/wiki/LayerParams) of the linked project seems to no longer work -- is there a mirror available anywhere?

**Comment by lauphedo** [comment]
Yeah, it would be nice if there was a mirror. I haven't seen a mirror yet, unfortunately.
